libdino/src/service/database.vala

```diff
@@ -17,10 +17,11 @@ public class Database : Qlite.Database {
         public Column<bool> enabled = new Column.BoolInt("enabled");
         public Column<string> roster_version = new Column.Text("roster_version") { min_version=2 };
         public Column<long> mam_earliest_synced = new Column.Long("mam_earliest_synced") { min_version=4 };
+        public Column<string> certificate = new Column.Text("certificate");
```

Comment:
The version of the database should be bumped, with an appropriate “min_version” for the certificate column, otherwise, the database will be lacking the “certificate” column, leading to crashes.

Reply:
Indeed, testing the PR with a previous db yields `database.vala:127: SQLite error: 1 - no such column: certificate` errors.